LSP.sublime-settings

```diff
@@ -293,6 +293,15 @@
   //
   //     // Extra variables to override/add to language server's environment.
   //     "env": { },
+  //
+  //     // Sets the diagnostics mode:
+  //     // "open_files" - if the server supports `diagnosticProvider`, diagnostics
+  //     //     are requested only for files which are opened in the editor and they
+  //     //     are cleared when the file gets closed.
+  //     // "workspace" - if the server supports `diagnosticProvider.workspaceDiagnostics`,
+  //     //     diagnostics are requested for all files in the project folders. Diagnostics
+  //     //     reported for files not within the project folders are ignored.
+  //     "diagnostics_mode": "open_files",
```

Comment:
I could switch the two sentences for the "workspace" option. Then it would be more clear that those sentences are intependent. I'm not sure whether it is important to mention that the filtering only applies if there are folders in the sidebar. I mean if there are no folders, then there is not really anything you could use as a filter, so this is kind of self-explanatory.

Reply:
Maybe start description of the option with a sentence like "When project has folders the this option specifies whether diagnostics are ignored and when. Otherwise it has no effect."

Reply:
Do you agree with this suggestion - adding initial sentence that clarifies that this option is only relevant when there are project folders?

Reply:
Yes. If you think another rewording would be better, feel free to adjust and push on the branch.
But if there are no project folders, the setting might still have an effect (in case of pull diagnostics).